test-suite/tests/compact-p002-out.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@context": {
+    "ex": {"@id": "http://example.org/"}
+  },
+  "@id": "http://example.org/id1",
+  "@type": ["http://example.org/Type1", "http://example.org/Type2"],
+  "http://example.org/term": {"@id": "http://example.org/id2"}
+}
```

Comment:
As the text above is written I believe you are correct. But I've actually argued below that we shouldn't do this. I think we should specify one way to create prefixes (append a colon (`:`) to your term). I now see that you argued for this over on #469.

I also see @gkellogg's counter argument:

https://github.com/json-ld/json-ld.org/issues/469#issuecomment-285713327

However, don't these contexts need to change to call for JSON-LD 1.1 processing rules to be applied anyway? If so, why not make the simpler change?